w3d_translator/w3d_translator.py

```diff
@@ -40,7 +40,7 @@ def create_out(out_dir: Path, force: bool):
         out_dir.mkdir()
     except PermissionError:
         # TODO 54: Catch error when files are being used in another process
-        # TODO: Retry a certain number of times?
+        # TODO 54: Retry a certain number of times?
```

Comment:
Bleh. I hate retry logic. Maybe you have more patience than I do!

Reply:
Yeah it's very weird. It seems like Unity is doing some stuff under the hood for a little while even after the GUI closes? So I've had times where I re-run the script and it's still locked out from the previous try

Reply:
oooo interesting. I guess retry is the right thing to do here. That stinks though!